### PR TITLE
EuroPython 2020 talk proposal

### PR DESCRIPTION
**Pull request by sgibson91**
This PR adds a proposal for a talk on Binder at EuroPython 2020.

Addresses #4

### CODE REVIEW

File: conferences/abstracts/EuroPython2020-talk.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+## Abstract (long)
+
+_At most 1500 characters. Current count (with spaces): 1182_
+
+As reproducibility gains traction in the data science community, the need to package code, data and the computational environment is growing.
+
+There are many tools that address different aspects of this type of packaging, such as Jupyter Notebooks for literate programming, Docker for containerising and porting computational environments, and so on.
+But they represent barriers to reproducibility as each one requires time and effort to learn.
+
+Project Binder provides the tooling for generating reproducible computational analyses with a web-based interface, such that users do not have to worry about all the moving parts so long as they have followed software best practices: their code is version controlled and they've captured the dependencies the analysis needs to run.
```
Comment (thobson88) [inline conferences/abstracts/EuroPython2020-talk.md:28]: Suggest: "...so long as they have followed basic software development best practices: ..." (i.e. adding the word "basic", so it sounds less onerous).

File: conferences/abstracts/EuroPython2020-talk.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Binder then allows the users to easily share their analyses by hosting the compute in the cloud and providing a unique URL to the code repository.
+
+During this talk, Sarah will introduce Binder (the service), BinderHub (the technological infrastructure) and mybinder.org (a public instance of a Binder service, free for anyone to use) and demonstrate how it can be used to share Python environments and analyses.
+
+## Abstract (short)
+
+_At most 500 characters. Current count (with spaces): 441_
+
+Packaging code, data and computational environments to provide reproducible analyses requires extra time and effort to learn the required tools.
+Project Binder provides open-source tooling for executing reproducible analyses in the cloud, and enabling them to be shared via a single, unique URL.
```
Comment (thobson88) [inline conferences/abstracts/EuroPython2020-talk.md:38]: Maybe add at the end of this sentence something like: "... unique URL, without imposing any additional overhead on the analyst."

File: conferences/abstracts/EuroPython2020-talk.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Subtitle
+
+(_Optional_)
+
+## Abstract (long)
+
+_At most 1500 characters. Current count (with spaces): 1182_
+
+As reproducibility gains traction in the data science community, the need to package code, data and the computational environment is growing.
```
Comment (DavidBeavan) [inline conferences/abstracts/EuroPython2020-talk.md:23]: You could successfully argue it's not just growing in data science, but in all (scholarly) research. That might widen your audience, who may be making their go/no-go decision on the first para.

File: conferences/abstracts/EuroPython2020-talk.md (hunk 1 above)
Comment (DavidBeavan) [inline conferences/abstracts/EuroPython2020-talk.md:28]: 'tooling': You could name-drop the core components Binder uses somewhere in here, e.g. jupyter, docker... or make the connection to them in the para above

File: conferences/abstracts/EuroPython2020-talk.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+There are many tools that address different aspects of this type of packaging, such as Jupyter Notebooks for literate programming, Docker for containerising and porting computational environments, and so on.
+But they represent barriers to reproducibility as each one requires time and effort to learn.
+
+Project Binder provides the tooling for generating reproducible computational analyses with a web-based interface, such that users do not have to worry about all the moving parts so long as they have followed software best practices: their code is version controlled and they've captured the dependencies the analysis needs to run.
+Binder then allows the users to easily share their analyses by hosting the compute in the cloud and providing a unique URL to the code repository.
```
Comment (DavidBeavan) [inline conferences/abstracts/EuroPython2020-talk.md:29]: Can you be clearer about 'users', perhaps naively I see two groups a) users who have a repo and Binderise (is that a word?) it b) users who interact with the hosted Binder instance

File: conferences/abstracts/EuroPython2020-talk.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+During this talk, Sarah will introduce Binder (the service), BinderHub (the technological infrastructure) and mybinder.org (a public instance of a Binder service, free for anyone to use) and demonstrate how it can be used to share Python environments and analyses.
+
```
Comment (DavidBeavan) [inline conferences/abstracts/EuroPython2020-talk.md:32]: Great, does the audience need reminding why reproducibility is worthwhile? Even if Binder is easier, it's still some effort. Can you give the Why bother?

File: conferences/abstracts/EuroPython2020-talk.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+- Best Practice
+- Data Science
+- Jupyter
+- Open-Source
+- Public Cloud (AWS/Google/...)
+
+## Type
+
+Talk (30 mins)
```
Comment (DavidBeavan) [inline conferences/abstracts/EuroPython2020-talk.md:66]: What's your Q&A strategy (inline or at the end)? The call makes specific reference to Q&A time

Comment (sgibson91) [reply conferences/abstracts/EuroPython2020-talk.md:66]: Uhh, my Q&A strategy is usually wing it 😬 Sorry, converting the form into Markdown hasn't made this very clear but this is a dropdown box selection.

File: conferences/abstracts/EuroPython2020-talk.md (hunk 4 above)
Comment (sgibson91) [reply conferences/abstracts/EuroPython2020-talk.md:29]: "code creators"?

Comment (DavidBeavan) [reply conferences/abstracts/EuroPython2020-talk.md:29]: Y

**Comment by sgibson91** [comment]
Submitted and merging! :tada:

**Pull request merged**